dev/docker/test.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+docker run -it --rm\
+    -v $(cd ../../; pwd):/root/workspace/Chaos\
+    -p 8082:80\
+    -p 8081:8081\
+    chaos\
+    python -m unittest
```

Comment:
No @phil-r is right it does need quotes. The argument is intended for `unittest`, without the quotes bash tries to expand it and fails.